src/components/sections/pricing/plans/components/card/index.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import {
+  StyledPriceCardFeature,
+  StyledPriceCardFeatureItemLI,
+  StyledPriceCardFeatureItemsWrapper,
+  StyledPriceCardPrice,
+  StyledPricingCardButtonWrapper,
+  StyledPricingCardPriceWrapper,
+  StyledPricingCardWrapper,
+} from './card.styles';
+import type { PricingCardProps } from './card.types';
+
+import { Button } from '@/components/elements/button';
+import { CheckIcon } from '@/components/elements/check-icon';
+import { Text } from '@/components/elements/text';
+import type { TextProps } from '@/components/elements/text/text.types';
+import { DownloadButton } from '@/components/modules/download-button';
+
+const renderFeatures = (
+  feature: PricingCardProps['features'][number],
+  $isHighlighted: PricingCardProps['$isHighlighted'],
+) => {
+  const textColor: TextProps['color'] = $isHighlighted ? 'white' : 'lightGrey';
+  const itemColor: TextProps['color'] = $isHighlighted ? 'white' : 'darkGrey';
+
+  return (
+    <StyledPriceCardFeature>
+      <Text fontSize="m" fontWeight="medium" color={textColor}>
+        {feature.text}
+      </Text>
+      <StyledPriceCardFeatureItemsWrapper>
+        {Array.isArray(feature.items) &&
+          feature.items?.length > 0 &&
+          feature.items.map(item => (
+            <StyledPriceCardFeatureItemLI>
+              <CheckIcon />
+              <Text fontSize="m" color={itemColor}>
+                {item}
+              </Text>
+            </StyledPriceCardFeatureItemLI>
+          ))}
+      </StyledPriceCardFeatureItemsWrapper>
+    </StyledPriceCardFeature>
+  );
+};
+
+export const PricingCard = ({
+  title,
+  price,
+  priceDescription,
+  isDownload,
+  CTA,
+  features,
+  $isHighlighted,
+}: PricingCardProps) => {
+  const TextColor: TextProps['color'] = $isHighlighted ? 'white' : 'lightGrey';
+  return (
+    <StyledPricingCardWrapper $isHighlighted={$isHighlighted}>
+      <StyledPricingCardPriceWrapper>
+        <Text fontSize="l" color={TextColor}>
+          {title}
+        </Text>
+        <Text fontSize="l" color="white">
+          <StyledPriceCardPrice>{price === 0 ? 'Free' : `€${price}`}</StyledPriceCardPrice>
```

Comment:
Could you leave a TODO, that this will need a small refactor and will be a dynamic value when working on the auth integration?